src/pepr/operator/controllers/keycloak/authservice/authservice.ts

```diff
@@ -33,35 +34,58 @@ export async function authservice(pkg: UDSPackage, clients: Map<string, Client>)
     pkg.spec?.sso || [],
   );
 
+  // Check if we're in ambient mode by looking for the waypoint annotation
+  const isAmbient =
+    pkg.metadata?.annotations?.["istio.io/use-waypoint"] !== undefined ||
+    pkg.spec?.network?.serviceMesh?.mode === "ambient";
```

Comment:
nit: silly q - is there a reason why you can't compare the actual enum like so?
```suggestion
    pkg.spec?.network?.serviceMesh?.mode === Mode.Ambient;
```

Reply:
OBE